.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+## :memo: PR Checklist
+
+Selecione quais requisitos o seu PR atende:
+
+- [ ] Os Commits seguem as guidelines: https://github.com/BrasilAPI/BrasilAPI/blob/master/CONTRIBUTING.md
+- [ ] Adicionei testes para as mudanças implementadas (bug fixes / features)
+- [ ] Adicionei documentação para as mudanças implementadas (bug fixes / features)
+
+
+## :information_source: Sobre o PR
+Que tipo de mudança esse PR propõe?
+
+<!-- selecione os items que se aplicam usando "x". -->
+```
```

Comment:
Aqui você adicionou uma checklist, mas dentro de um bloco de código, assim a checklist não funcionará

É essa a intenção mesmo?

Reply:
@murilohns Sim sim intencional, por isso o descritivo "marque com um X"... O template está baseado no utilizado pelo pessoal do nestjs, mas podemos mudar este comportamento se preferir

Reply:
@murilohns Entendo! tb não sou fã de burocracia haha ...pensei mais em algo pra gente ter um summary do PR antes de descer nas implementações.... algo menos burocrativo faz sentido sim!

Reply:
Deixa eu entrar de bico aqui rsrsrs
Bem, eu acho muito legal quando existe template para PR bem completo mas que seja "obrigatório" segui-lo.
Eu gosto de um template pois eles nos guiam de como descrever bem o que está sendo implementado, e acho isso bem importante. Mas se eu for forçado a preencher um "template gigante" pra qualquer coisa eu vou desistir de do PR rsrs

Acho que é isso. Um template super completo como guia, mas nada obrigatório. Seja qual for a forma o importante é se fazer entendível.

ps: Tipo esse PR, 1 frase que explica o objetivo. Talvez seria legal complementar com uma foto de como é o template :p

Reply:
Pessoal, isso não seria questão de burocracia, mas um guia para auxiliar os futuros contribuidores. Mais fácil saberem tudo que é preciso vendo o checklist do que tendo que alguém parar e sugerir revisão.

Reply:
Eu sugiro remover do bloco de código, pois já que a pessoa precisa marcar um X as opções precisam estar disponíveis. https://github.com/BrasilAPI/BrasilAPI/pull/272#discussion_r645930032

Reply:
Pessoal, minha sugestão é remover todo esse bloco de código com checks , não é algo que acrescenta muita informação para quem irá revisar, poisos as mensagens de commit bem detalhadas já conterão essas informações, bem como a explicação feita sobre o que o PR se propõe.

Reply:
Dada a discussão entendi que o melhor seria momentaneamente remover esse trecho pra simplificar e caso sentimos a necessidade podemos adicionar novamente.